setup.py

```diff
@@ -63,6 +64,49 @@ def clean_optional_plugins(self):
             os.chdir(root_dir)
 
 
+class SimpleCommand(Command):
+    """Make Command implementation simpler."""
+
+    user_options = []
+
+    def __init__(self, *args, **kwargs):
+        """Store arguments so it's possible to call other commands later."""
+        super().__init__(*args, **kwargs)
+        self.__args = args
+        self.__kwargs = kwargs
+
+    @abstractmethod
+    def run(self):
+        """Run when command is invoked."""
+
+    def run_command(self, command_class):
+        """Run another command with same __init__ arguments."""
+        command_class(*self.__args, **self.__kwargs).run()
+
+    def initialize_options(self):
+        """Set default values for options."""
+
+    def finalize_options(self):
+        """Post-process options."""
+
+
+class Linter(SimpleCommand):
+    """Lint Python source code."""
+
+    description = 'Run logical, stylistic, analytical and formatter checks.'
+
+    def run(self):
+        try:
+            check_call('selftests/signedoff-check.sh', shell=True)
+            check_call('selftests/spell.sh', shell=True)
+            check_call('selftests/inspekt-indent.sh', shell=True)
+            check_call('selftests/inspekt-style.sh', shell=True)
+            check_call('selftests/lint.sh', shell=True)
+        except CalledProcessError as e:
+            print("Failed during lint checks: ", e)
+            sys.exit(-1)
```

Comment:
Positive exit codes are preferable.

Reply:
I tryied to follow the `AVOCADO_GENERIC_CRASH` that is returning -1. Also, `avocado/core/main.py` has some references to `-1` returns. Yes, I know that this is not an Avocado failure, but it was just a generic choice. Do you have any number/pattern suggestions here?

Reply:
> I tried to follow the `AVOCADO_GENERIC_CRASH` that is returning -1. Also, `avocado/core/main.py` has some references to `-1` returns. Yes, I know that this is not an Avocado failure, but it was just a generic choice. Do you have any number/pattern suggestions here?

That's also not a good value, IMO.  And just to make it clear, I may be guilty of some of those, because if I lower my guard, I'll probably resort to doing `sys.exit(-1)` too.  In fact, I remember I someone noticed this in code I've written recently.

Anyway, this supports the idea of positive integers only:  https://www.gnu.org/software/libc/manual/html_node/Exit-Status.html

And there are also range limits because the exist status is encoded into the overall process termination status.